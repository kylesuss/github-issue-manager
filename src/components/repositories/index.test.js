import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Repositories from './index'

test('it renders', () => {
  const wrapper = shallow(<Repositories />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
