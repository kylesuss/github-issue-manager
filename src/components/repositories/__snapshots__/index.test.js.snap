// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div>
  <ConnectFunction />
  <Switch>
    <Route
      exact={true}
      path="/repositories"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/repositories/:id"
    />
    <Redirect
      to="/repositories"
    />
  </Switch>
</div>
`;
